### DISCUSSION TITLE
huh

### DISCUSSION THREAD

**Discussion by Maxxy1231sad** [3kh0/ext-remover#858] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
"Help me! I'm an idiot!"
"I had far too much faith in society when making this page. Some of you skids out there are really, really stupid and also can't read. So here are the answers to some commonly asked questions."

"How do I get an extension ID?"

"Okay, fair. Extension IDs are leaked in a couple of places. Generally, the **best way to get them is to go to extension settings and copy the URL query value.**"

I bet they're really smort enough to understand what that means 3kh0 :)

**Comment by rwg100** [comment]
"help me I'm retarded what's a skid"

**Reply by Aka-but-nice** [reply]
Not necessarily, in other words, it's just someone who knows nothing about technology, a technological illiterate if you will.

**Reply by Maxxy1231sad** [reply]
erm.. no I dont steal code and claim it- Just tryna get through the days like everyone else-

**Comment by Epicminer256** [comment]
I don't think insulting users of your code will make them want to use it. I cannot keep my respect for a dev who insults their users.

Realize that the audience and users you are working with don't know anything about computers, so try your best to help them. They will better respect your projects and you as a developer.

**Reply by Aka-but-nice** [reply]
Just like I said, most developers have no problem with people using their programs, it's the fact how they insist on knowing something when they do not. Again, a **technological illiterate** like I said.

**Reply by Maxxy1231sad** [reply]
Yall took this so literal tho-

**Reply by Maxxy1231sad** [reply]
yall need to chill :p

**Comment by tuta-amb** [comment]
you balls need to calm your balls down

**Reply by Maxxy1231sad** [reply]
balls balls goose

**Comment by Ender-Ghost-Craft** [comment]
bro I swear how are these left bellcurve 6th graders finding this

**Reply by Maxxy1231sad** [reply]
goose
